InternalMessage result
ExceptionCallback<Void,Exception> exceptionCallback
